packages/core/src/node/tab-indent.ts

```diff
@@ -0,0 +1,41 @@
+import type { NodeSpec, NodeType } from 'prosemirror-model';
+import type { Keymap } from 'prosemirror-commands';
+import { Fragment } from 'prosemirror-model';
+import { EditorState } from 'prosemirror-state';
+
+import { Node } from '../abstract';
+import { SerializerNode } from '../serializer/types';
+
+export class TabIndent extends Node {
+    name = 'tab_indent';
+    schema: NodeSpec = {
+        group: 'inline',
+        inline: true,
+        selectable: false,
+        parseDOM: [{ tag: `span[class='tab-indent']` }],
+        toDOM: () => ['span', { class: 'tab-indent', style: 'letter-spacing: 20px; display: inline-block' }, ' '],
```

Comment:
Style should be moved into the theme package.

Reply:
ok